conf/machine/imx7d-remarkable2.conf

```diff
@@ -27,13 +27,8 @@ MACHINE_ESSENTIAL_EXTRA_RDEPENDS += " \
     u-boot-remarkable2 \
 "
 
-# The Linux kernel expects firmware for bcm43456,
-# but there isn't yet support upstream for bcm43456 so we also
-# include bcm43455 as a fallback.
 MACHINE_EXTRA_RRECOMMENDS += " \
   linux-firmware-imx-sdma-imx7d \
-  linux-firmware-bcm43455 \
-  linux-firmware-bcm43456 \
```

Comment:
Only remove the linux-firmware-bcm43456 and keep the rest so user can find the info